[2023-04-09T00:34:51.618+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: api_dag.is_api_available manual__2023-04-09T00:29:48.109385+00:00 [queued]>
[2023-04-09T00:34:51.628+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: api_dag.is_api_available manual__2023-04-09T00:29:48.109385+00:00 [queued]>
[2023-04-09T00:34:51.629+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-04-09T00:34:51.630+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2023-04-09T00:34:51.631+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-04-09T00:34:51.639+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-04-09 00:29:48.109385+00:00
[2023-04-09T00:34:51.646+0000] {standard_task_runner.py:55} INFO - Started process 9789 to run task
[2023-04-09T00:34:51.650+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'api_dag', 'is_api_available', 'manual__2023-04-09T00:29:48.109385+00:00', '--job-id', '68', '--raw', '--subdir', 'DAGS_FOLDER/api.py', '--cfg-path', '/tmp/tmpgl2cyhf4']
[2023-04-09T00:34:51.653+0000] {standard_task_runner.py:83} INFO - Job 68: Subtask is_api_available
[2023-04-09T00:34:51.728+0000] {task_command.py:376} INFO - Running <TaskInstance: api_dag.is_api_available manual__2023-04-09T00:29:48.109385+00:00 [running]> on host c12a518d534c
[2023-04-09T00:34:51.782+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=api_dag
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-04-09T00:29:48.109385+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-09T00:29:48.109385+00:00
[2023-04-09T00:34:51.784+0000] {http.py:120} INFO - Poking: api/
[2023-04-09T00:34:51.794+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2023-04-09T00:34:51.796+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-04-09T00:34:52.042+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-04-09T00:34:52.065+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=api_dag, task_id=is_api_available, execution_date=20230409T002948, start_date=20230409T003451, end_date=20230409T003452
[2023-04-09T00:34:52.108+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-04-09T00:34:52.134+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
